docs/hub/endpoints.md

```diff
@@ -0,0 +1,40 @@
+| -------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| /api/models  <br/>  GET                                                          | Get information from all models in the Hub. You can specify additional parameters to have more specific results.  <br/>- filter: Filter based on tags, such as `text-classification` or `spacy`.<br/>- sort: Property to use when sorting. <br/>- direction: Direction in which to sort. <br/>- limit: Limit the number of models fetched. <br/>- full: Whether to fetch most model data, such as all tags, the files, etc. <br/>- config: Whether to also fetch the repo config. | `list_models()`                        | ```params= {   filter:filter, full: full,    sort: sort,   direction: direction,   limit: limit,   config: config }```                   |
+| /api/models/{repo_id} <br/> /api/models/{repo_id}/revision/{revision} <br/>  GET | Get all information for a specific model.                                                                                                                                                                                                                                                                                                                                                                                                                                         | `model_info(repo_id, revision)`        | ```headers = { authorization :  "Bearer $token" }```                                                                                     |
+| /api/repos/ls <br/>  GET                                                         | Get list of all stored files for user or organization.                                                                                                                                                                                                                                                                                                                                                                                                                               | `list_repos_objs(token, organization)` | ```headers = { authorization :  "Bearer $token" }``` <br/>```params= {   organization:organization}```                                   |
```

Comment:
WDYT of putting the placeholders such as `authorization` between quotes so that it's simpler to copy/paste? Right now it's formatted as JS, but it's invalid JSON

Reply:
Good idea, done!